src/reference/snakemake-style-guide.rst

```diff
@@ -0,0 +1,41 @@
+=====================
+Snakemake style guide
+=====================
+
+When in doubt, refer to `Snakemake's own best practices
+guide <https://snakemake.readthedocs.io/en/stable/snakefiles/best_practices.html>`__.
+
+-  Avoid ``run`` blocks; implement custom Python in scripts called in a
+   ``shell`` block.
```

Comment:
_non-blocking_

I think each recommendation should be a heading, not a top-level list item. This makes it stand out more, be linkable, makes a proper local TOC possible, encourages the recommendation to be "snappy", etc.

For example, see how I wrote a similar style guide back in 2018: https://github.com/blab/styleguide/blob/6aa5d7aa42acfa97e57a5ee05a4175f158502cac/nextstrain-builds.md

Reply:
Done: 57fe188819ef9e614e9d9641ddfd2b1140c713bb